pkg/v1/tkg/tkgconfigbom/bom_test.go

```diff
@@ -246,6 +246,34 @@ var (
 					Expect(err).ToNot(HaveOccurred())
 					Expect(bomConfiguration).ToNot(BeNil())
 				})
+				It("Should return the correct etcd configuration", func() {
+					Expect(err).ToNot(HaveOccurred())
+					Expect(bomConfiguration.KubeadmConfigSpec.Etcd.Local.DataDir).To(Equal("/var/lib/etcd"))
+					Expect(bomConfiguration.KubeadmConfigSpec.Etcd.Local.ImageRepository).To(Equal("registry.tkg.vmware.run"))
+					Expect(bomConfiguration.KubeadmConfigSpec.Etcd.Local.ImageTag).To(Equal("v3.4.13_vmware.4"))
+					Expect(len(bomConfiguration.KubeadmConfigSpec.Etcd.Local.ExtraArgs)).To(Equal(1))
+					Expect(bomConfiguration.KubeadmConfigSpec.Etcd.Local.ExtraArgs["experimental-initial-corrupt-check"]).To(Equal("true"))
```

Comment:
know it is just unit tests, but is this reflecting the opposite of reality.
iow, we shouldn't be expecting boms using pre-3.5 etcd to include the experimental-initial-corrupt-check, right?

Reply:
Yes. Updated to use fake arguments.